### PR TITLE
Upgrade to Cordova 5.0.0

### PR DESCRIPTION
**Pull request by bhariharan**

### CODE REVIEW

File: libs/SalesforceHybrid/res/xml/config.xml [hunk 1]
```diff
@@ -20,6 +20,6 @@
     <feature name="com.salesforce.oauth"><param name="android-package" value="com.salesforce.androidsdk.phonegap.plugin.SalesforceOAuthPlugin"/></feature>
     <feature name="com.salesforce.sfaccountmanager"><param name="android-package" value="com.salesforce.androidsdk.phonegap.plugin.SFAccountManagerPlugin"/></feature>
     <feature name="com.salesforce.sdkinfo"><param name="android-package" value="com.salesforce.androidsdk.phonegap.plugin.SDKInfoPlugin"/></feature>
-    <feature name="com.salesforce.smartstore"><param name="android-package" value="com.salesforce.androidsdk.smartstore.phonegap.plugin.SmartStorePlugin"/></feature>
-    <feature name="com.salesforce.smartsync"><param name="android-package" value="com.salesforce.androidsdk.smartsync.phonegap.plugin.SmartSyncPlugin"/></feature>
+    <feature name="com.salesforce.smartstore"><param name="android-package" value="com.salesforce.androidsdk.phonegap.plugin.SmartStorePlugin"/></feature>
```
Comment (bhariharan) [inline libs/SalesforceHybrid/res/xml/config.xml]: This is an unrelated bugfix (we forgot to update the package names here when we moved to ```SalesforceHybrid```).

File: libs/test/SalesforceHybridTest/res/xml/config.xml [hunk 2]
```diff
@@ -20,8 +20,8 @@
     <feature name="com.salesforce.oauth"><param name="android-package" value="com.salesforce.androidsdk.phonegap.plugin.SalesforceOAuthPlugin"/></feature>
     <feature name="com.salesforce.sfaccountmanager"><param name="android-package" value="com.salesforce.androidsdk.phonegap.plugin.SFAccountManagerPlugin"/></feature>
     <feature name="com.salesforce.sdkinfo"><param name="android-package" value="com.salesforce.androidsdk.phonegap.plugin.SDKInfoPlugin"/></feature>
-    <feature name="com.salesforce.smartstore"><param name="android-package" value="com.salesforce.androidsdk.smartstore.phonegap.plugin.SmartStorePlugin"/></feature>
-    <feature name="com.salesforce.smartsync"><param name="android-package" value="com.salesforce.androidsdk.smartsync.phonegap.plugin.SmartSyncPlugin"/></feature>
+    <feature name="com.salesforce.smartstore"><param name="android-package" value="com.salesforce.androidsdk.phonegap.plugin.SmartStorePlugin"/></feature>
```
Comment (bhariharan) [inline libs/test/SalesforceHybridTest/res/xml/config.xml]: Same here - bug fix.

**Comment by wmathurin** [comment]
So it just works?

**Comment by bhariharan** [comment]
Yep - it just works! They added new APIs for runtime permissions, so plugins can get callbacks and act accordingly. They also cleaned up their Gradle scripts a bit, but otherwise no major overhaul. I'll need to tweak our postinstall script, but that should be easy.

**Comment by wmathurin** [comment]
Great. :+1:

**Comment by kchitalia** [comment]
:+1:

**Comment by bhariharan** [comment]
Updated ```forcedroid``` [here](https://github.com/forcedotcom/SalesforceMobileSDK-Android/commit/73b820addcf19159f6950cfb14d5c0f93a9d77d7).
